freno-client.gemspec

```
@@ -20,5 +20,5 @@ Gem::Specification.new do |spec|
 
   spec.files = `git ls-files -z`.split("\x0").grep_v(/^(bin|test)/)
 
-  spec.add_dependency "faraday", "~> 0"
+  spec.add_dependency "faraday", ">= 0", "< 3"
```

Could we drop the `>= 0` condition?

```suggestion
  spec.add_dependency "faraday", "< 3"
```